$OpenBSD: patch-lib_util_c,v 1.1 2012/04/20 20:54:17 ajacoutot Exp $

util.c:284: error: 'UINT32_MAX' undeclared (first use in this function)

--- lib/util.c.orig	Fri Apr 20 22:38:29 2012
+++ lib/util.c	Fri Apr 20 22:39:23 2012
@@ -45,6 +45,7 @@
 #include <config.h>
 
 #include <stdio.h>
+#include <stdint.h>
 #include <ctype.h>
 #include <errno.h>
 #include <grp.h>
